package.json

```diff
@@ -9,18 +9,18 @@
   "scripts": {
     "postinstall": "npx patch-package && cd desktop && npm install",
     "clean": "npx react-native clean-project-auto",
-    "android": "scripts/set-pusher-suffix.sh &&  concurrently \"npx react-native run-android --port=8083\" npm:prettier-watch",
-    "ios": "scripts/set-pusher-suffix.sh && concurrently \"npx react-native run-ios --port=8082\" npm:prettier-watch",
-    "ipad": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (12.9-inch) (4th generation)\"\" npm:prettier-watch",
-    "ipad-sm": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (9.7-inch)\"\" npm:prettier-watch",
+    "android": "scripts/set-pusher-suffix.sh && npx react-native run-android --port=8083",
```

Comment:
Was `concurrently` in all these not needed or what?

Reply:
I think we need `concurrently` for this to work on Windows.. But let me find the commit

Reply:
Concurrently was necessary for when two long-running processes have to run at once. Since there are no longer multiple long-running processes, concurrently can be removed. It's still used in a couple of scripts though like the web script which needs to have both the web proxy and the web server running.